Apps/W1/Shopify/app/src/Products/Reports/ShpfyAddItemAsVariant.Report.al

```diff
@@ -0,0 +1,116 @@
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Inventory.Item;
+
+/// <summary>
+/// A report to add items as variants to a parent product.
+/// </summary>
+report 30117 "Shpfy Add Item As Variant"
```

Comment:
I know that the other reports don't have any AccessByPermission property set, but I think it would be useful to hide this report in the UI  from Users with missing permissions.

I would try to set the AccessByPermission to e.g. `tabledata "Shpfy Variant" = RIMD` (this was the table that I found in the code)

Reply:
Good idea. I'll run it by Andrei.